api/src/migrations/api/1619711501000-DropNotNullsOnTimeUserMetadataColumns.ts

```diff
@@ -0,0 +1,50 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { v4 } from 'uuid';
+
+export class DropNotNullsOnTimeUserMetadataColumns1619711501000
+  implements MigrationInterface {
+  private tablesToAlter = ['organizations', 'projects', 'scenarios'];
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await Promise.all(
+      this.tablesToAlter.map(async (table) => {
+        await queryRunner.query(`
+ALTER TABLE ${table}
+  ALTER COLUMN created_at DROP NOT NULL,
+  ALTER COLUMN last_modified_at DROP NOT NULL,
+  ALTER COLUMN created_by DROP NOT NULL;
+`);
+      }),
+    );
```

Comment:
[minor/style]
```
await Promise.all(
      this.tablesToAlter.map((table) =>
        queryRunner.query(`
ALTER TABLE ${table}
ALTER COLUMN created_at DROP NOT NULL,
ALTER COLUMN last_modified_at DROP NOT NULL,
ALTER COLUMN created_by DROP NOT NULL;`),
      ),
    );
```

Reply:
done in ac8f10379e509cce7ad6a530755ad19711d5e2e9